CONTRIBUTING.md

```diff
@@ -1,10 +1,15 @@
 # Contributing
 
-## Updating CHANGELOG.md
+## CHANGELOG 
+
+Authors are expected to include a changelog file with their pull requests. The changelog file
+should be a new file created in the `changelogs/unreleased` folder. The file should follow the
+naming convention of `issue-username` and the contents of the file should be your text for the
```

Comment:
issue-username or pr-username?

Reply:
So looking at past changelogs we seem to equally use both. I have no strong opinions on it.

Reply:
It might be nice to keep track of issues fixed, but I'd start with pr-username.

Reply:
+1 for pr-username